static/js/compose.js

```diff
@@ -696,6 +733,11 @@ function validate_stream_message() {
         }
     }
 
+    if (user_acknowledged_all_everyone === false) {
+        compose_error("Please remove @all / @everyone or acknowledge that you will be spamming everyone!");
```

Comment:
Yes, I'd like to simplify this string too. It's a bit long and wordy. I'm working on adding support so that when you actually remove @all, the error goes away.